src/q7/powerelec-ELEC2660/exam/2019/Janvier/All/powerelec-ELEC2660-exam-2019-Janvier-All.tex

```diff
@@ -0,0 +1,88 @@
+\documentclass[en]{../../../../../../eplexam}
```

Comment:
Shouldn't it be fr then?

Reply:
Bah, c'est quand même un cours en anglais, donc j'aime autant le laisser comme ça pour inciter les gens à faire une correction en anglais si possible. Et de base, j'allais peut-être le traduire, mais les vacances plutôt "restreintes" qu'on va hériter ne me motivent plus à le faire. :smile: